From 93772c511d954d755cc9128c58ed8968a5af541b Mon Sep 17 00:00:00 2001
From: "Mingde (Matthew) Zeng" <matthewzmd@gmail.com>
Date: Wed, 29 Jul 2020 08:47:09 -0400
Subject: [PATCH] Remove OOM tests from runtest/mm

Disable OOM tests, as they might cause oeqa ssh connection lost

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Mingde (Matthew) Zeng <matthew.zeng@windriver.com>

---
 runtest/mm | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/runtest/mm b/runtest/mm
index 481d39691..cf207d10e 100644
--- a/runtest/mm
+++ b/runtest/mm
@@ -74,12 +74,6 @@ ksm06_2 ksm06 -n 10000
 
 cpuset01 cpuset01
 
-oom01 oom01
-oom02 oom02
-oom03 oom03
-oom04 oom04
-oom05 oom05
-
 swapping01 swapping01 -i 5
 
 thp01 thp01 -I 120
